src/main/resources/db/migration/V1_42__ADD_NEW_WEST_MIDLANDS_TEAMS.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO team (code, name, ldu_code) VALUES ('N52MAG', 'Multi-Agency Gang Unit (MAGU)', 'N52BIRL');
+INSERT INTO team (code, name, ldu_code) VALUES ('N52AL2', 'BCCF Pending Allocations Central', 'N52BIRL');
+INSERT INTO team (code, name, ldu_code) VALUES ('N52AL4', 'BCCF Pending Allocations East', 'N52BIRL');
+INSERT INTO team (code, name, ldu_code) VALUES ('N52AL3', 'BCCF Pending Allocations North', 'N52BIRL');
+INSERT INTO team (code, name, ldu_code) VALUES ('N52AL1', 'BCCF Pending Allocations South', 'N52BIRL');
+INSERT INTO team (code, name, ldu_code) VALUES ('N52BlO', 'Birmingham Integrated Offender Management', 'N52BIRL');
+INSERT INTO team (code, name, ldu_code) VALUES ('N52BlS', 'Birmingham Integrated Offender Management', 'N52BIRL');
```

Comment:
Just checking, are these two teams with the same name?

Reply:
Aww good spot, thanks for picking that up 🙂 the same team name was put on the spreadsheet and I hadn't noticed 🙌